 CMakeLists.txt          | 8 ++++++++
 simulation/libsimulator | 2 +-
 2 files changed, 9 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 382b22c3b..3146c1775 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -491,6 +491,14 @@ elseif(MSVC)
 	)
 endif()
 
+option(ENABLE_MARCH_MTUNE_NATIVE "Use -march/mtune=native compiler flag." ON)
+if(ENABLE_MARCH_MTUNE_NATIVE AND CMAKE_CXX_COMPILER_ID MATCHES GNU AND CMAKE_BUILD_TYPE STREQUAL "Release")
+	add_compile_options(
+	    -march=native
+	    -mtune=native
+	)
+endif()
+
 if(static_runtime)
 	if (MSVC)
 		set(CMAKE_MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>")
diff --git a/simulation/libsimulator b/simulation/libsimulator
index da549dbfb..66ac6cc0b 160000
--- a/simulation/libsimulator
+++ b/simulation/libsimulator
@@ -1 +1 @@
-Subproject commit da549dbfb655e50b4d200ff91913b42794787cb4
+Subproject commit 66ac6cc0b248e4655b8d21004fd24e3b62b5908d
