 src/settings_pack.cpp | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/src/settings_pack.cpp b/src/settings_pack.cpp
index 939e5a46a..e801103a2 100644
--- a/src/settings_pack.cpp
+++ b/src/settings_pack.cpp
@@ -169,7 +169,7 @@ constexpr int CLOSE_FILE_INTERVAL = 0;
 		SET(no_atime_storage, true, nullptr),
 		SET(incoming_starts_queued_torrents, false, nullptr),
 		SET(report_true_downloaded, false, nullptr),
-		SET(strict_end_game_mode, true, nullptr),
+		SET(strict_end_game_mode, false, nullptr),
 		DEPRECATED_SET(broadcast_lsd, true, nullptr),
 		SET(enable_outgoing_utp, true, nullptr),
 		SET(enable_incoming_utp, true, nullptr),
@@ -223,7 +223,7 @@ constexpr int CLOSE_FILE_INTERVAL = 0;
 		SET(tracker_maximum_response_length, 1024*1024, nullptr),
 		SET(piece_timeout, 20, nullptr),
 		SET(request_timeout, 60, nullptr),
-		SET(request_queue_time, 3, nullptr),
+		SET(request_queue_time, 6, nullptr),
 		SET(max_allowed_in_request_queue, 500, nullptr),
 		SET(max_out_request_queue, 500, nullptr),
 		SET(whole_pieces_threshold, 20, nullptr),
@@ -233,7 +233,7 @@ constexpr int CLOSE_FILE_INTERVAL = 0;
 		SET(urlseed_wait_retry, 30, nullptr),
 		SET(file_pool_size, 40, nullptr),
 		SET(max_failcount, 3, &session_impl::update_max_failcount),
-		SET(min_reconnect_time, 60, nullptr),
+		SET(min_reconnect_time, 60, nullptr),
 		SET(peer_connect_timeout, 15, nullptr),
 		SET(connection_speed, 30, &session_impl::update_connection_speed),
 		SET(inactivity_timeout, 600, nullptr),
@@ -294,7 +294,7 @@ constexpr int CLOSE_FILE_INTERVAL = 0;
 		SET(increase_est_reciprocation_rate, 20, nullptr),
 		SET(decrease_est_reciprocation_rate, 3, nullptr),
 		SET(max_pex_peers, 50, nullptr),
-		SET(tick_interval, 500, nullptr),
+		SET(tick_interval, 250, nullptr),
 		SET(share_mode_target, 3, nullptr),
 		SET(upload_rate_limit, 0, &session_impl::update_upload_rate),
 		SET(download_rate_limit, 0, &session_impl::update_download_rate),
@@ -316,7 +316,7 @@ constexpr int CLOSE_FILE_INTERVAL = 0;
 		SET(utp_loss_multiplier, 50, nullptr),
 		SET(mixed_mode_algorithm, settings_pack::peer_proportional, nullptr),
 		SET(listen_queue_size, 5, nullptr),
-		SET(torrent_connect_boost, 30, nullptr),
+		SET(torrent_connect_boost, 60, nullptr),
 		SET(alert_queue_size, 1000, &session_impl::update_alert_queue_size),
 		SET(max_metadata_size, 3 * 1024 * 10240, nullptr),
 		DEPRECATED_SET(hashing_threads, 1, nullptr),
@@ -332,7 +332,7 @@ constexpr int CLOSE_FILE_INTERVAL = 0;
 		SET(peer_turnover, 4, nullptr),
 		SET(peer_turnover_cutoff, 90, nullptr),
 		SET(peer_turnover_interval, 300, nullptr),
-		SET(connect_seed_every_n_download, 10, nullptr),
+		SET(connect_seed_every_n_download, 5, nullptr),
 		SET(max_http_recv_buffer_size, 4*1024*204, nullptr),
 		SET(max_retry_port_bind, 10, nullptr),
 		SET(alert_mask, int(static_cast<std::uint32_t>(alert_category::error)), &session_impl::update_alert_mask),
@@ -352,7 +352,7 @@ constexpr int CLOSE_FILE_INTERVAL = 0;
 		SET(max_web_seed_connections, 3, nullptr),
 		SET(resolver_cache_timeout, 1200, &session_impl::update_resolver_cache_timeout),
 		SET(send_not_sent_low_watermark, 16384, nullptr),
-		SET(rate_choker_initial_threshold, 1024, nullptr),
+		SET(rate_choker_initial_threshold, 512, nullptr),
 		SET(upnp_lease_duration, 3600, nullptr),
 		SET(max_concurrent_http_announces, 50, nullptr),
 	}});
